ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/WebSocketTest.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.tests
+
+import io.ktor.client.features.websocket.*
+import io.ktor.client.tests.utils.*
+import io.ktor.http.cio.websocket.*
+import kotlin.test.*
+
+class WebSocketTest : ClientLoader() {
+
+    @Test
+    fun testEcho() = clientTests {
+        config {
+            install(WebSockets)
+        }
+
+        test { client ->
+            client.webSocket("ws://127.0.0.1:8080/websockets/echo") {
```

Comment:
Shouldn't we use `$TEST_SERVER` instead?

Reply:
`$TEST_SERVER` has a different protocol. I introduce a constant instead.